### PR TITLE
Disable TLS 1.0 and 1.1 in Nginx Ingress Controller for security reasons

### PR DESCRIPTION
**Pull request by yunho-microsoft**
Change tmpl file to disable TLS 1.0 and 1.1 in Nginx Ingress Controller. By default, it supports TLS 1.0, 1.1 and 1.2

### CODE REVIEW

**Review by sharmakhushboo — APPROVED** [review]

File: server/routerlicious/kubernetes/nginx/config/nginx.tmpl [hunk 1]
```diff
@@ -198,7 +198,7 @@ http {
     server_name_in_redirect off;
     port_in_redirect        off;
 
-    ssl_protocols {{ $cfg.SSLProtocols }};
```
Comment (hedasilv) [inline server/routerlicious/kubernetes/nginx/config/nginx.tmpl:201]: By the way @yunho-microsoft , where does .Cfg come from? Do we have to pass it manually when installing the nginx chart? We talked about the .tmpl file the other day but I couldn't figure out where the .Cfg is coming from.

**Review by hedasilv — APPROVED** [review]

**Comment by sharmakhushboo** [comment]
Could you drop a comment on https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/kubernetes/nginx/README.md to specify that the template now only supports tls1.2?

File: server/routerlicious/kubernetes/nginx/config/nginx.tmpl (hunk 1 above)
Comment (yunho-microsoft) [reply server/routerlicious/kubernetes/nginx/config/nginx.tmpl:201]: I haven't figured it out. We don't pass it manually. I think .Cfg means the default setting for Nginx but I don't find any documents about it. For the syntax of the tmpl file, this is the doc: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/custom-template/

From the doc, there is a link to the template file syntax: https://pkg.go.dev/text/template
I didn't spend much time on it yet.

**Review by tanviraumi — APPROVED** [review]

**Review by hedasilv — APPROVED** [review]

File: server/routerlicious/kubernetes/nginx/README.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ $ kubectl create configmap nginx-template --from-file=nginx.tmpl=./nginx.tmpl
 If your kube cluster uses rbac, deploy the roles and bindings from the rbac folder. 
 
 Deploy the controller now. This assumes a rbac enabled cluster, so remove <em>serviceAccountName: nginx-serviceaccount</em> line from the deployment files if your cluster is not rbac enabled.
+For security reasons, TLS 1.0 and 1.1 are disabled. Only TLS 1.2 is supported.
```
Comment (sharmakhushboo) [inline server/routerlicious/kubernetes/nginx/README.md:41]: Move this to the template section as that's where we are making the change?

**Comment by sharmakhushboo** [comment]
@tanviraumi just wondering if we need to add an entry to the breakingChange.md file?

File: server/routerlicious/kubernetes/nginx/README.md (hunk 2 above)
Comment (yunho-microsoft) [reply server/routerlicious/kubernetes/nginx/README.md:41]: Comments added

**Pull request merged**